src/raster/r.survey/r.survey.html

```diff
@@ -0,0 +1,290 @@
+
+<h2>INSTALLATION</h2>
+<h3>LINUX:</h3>
+
+1) Download the compressed folder of r.survey from <a href="https://doi.org/10.5281/zenodo.3993140">https://doi.org/10.5281/zenodo.3993140</a> and decompress it. 
+<p>
+2) Copy the file <em>r.survey.py</em> into the folder <em>/usr/lib/grass78/scripts/</em>
+<p> 
+3) Copy the file <em>r.survey.html</em> and the folder <em>r.survey_figures</em> into the directory <em>/usr/lib/grass78/docs/html/</em>
+<p> 
+4) Open GRASS and run the following commands in order to verify if it works: 
+<dd><div class="code"><pre>r.survey --h</pre></div></dd>
+<dd><div class="code"><pre>g.manual r.survey</pre></div></dd>
+
+<h3>WINDOWS:</h3>
+
+Depending on the type of GRASS installation carried out, the directory paths can slightly vary. 
+<p>
+1) Download the compressed folder of r.survey from <a href="https://doi.org/10.5281/zenodo.3993140">https://doi.org/10.5281/zenodo.3993140</a> and decompress it. 
+<p>
+2) Copy the file named <em>r.survey.bat</em> into the following directory:
+
+<dd><em>C:\Program Files\GRASS GIS 7.8\bin\</em>  (regular installation)</dd>
+<dd>or</dd>
+<dd><em>C:\OSGeo4W64\apps\grass\grass78\bin\</em> (installation from OSGeo)</dd>
+<p>
+3) Copy the file <em>r.survey.py</em> into the followin folder:
+
+<dd><em>C:\Program Files\GRASS GIS 7.8\scripts\</em>  (regular installation)</dd>
+<dd>or</dd>
+<dd><em>C:\OSGeo4W64\apps\grass\grass78\scripts\</em> (installation from OSGeo)</dd>
+<p>
+4) Copy the file <em>r.survey.html</em> and the folder <em>r.survey_figures</em> into the following directory:
+
+<dd><em>C:\Program Files\GRASS GIS 7.8\docs\html\</em>  (regular installation)</dd>
+<dd>or</dd>
+<dd><em>C:\OSGeo4W64\apps\grass\grass78\docs\html\</em> (installation from OSGeo)</dd>
+<p>
+
+5) Open GRASS and run the following commands in order to verify if it works: 
+<dd><div class="code"><pre>r.survey --h</pre></div></dd>
+<dd><div class="code"><pre>g.manual r.survey</pre></div></dd>
+
+<h2>NOTES</h2>
+The software was designed as a GRASS GIS python module whose source code was  written using the GRASS Scripting library and the pyGRASS library.
+<p>
+Multi-core processing is used by r.survey for reducing the computational time. When the aim is to derive the values of the visibility indexes along a given path (e.g. a road or a UAV track) viewpoints can be very dense in terms of number per unit of distance and the more the viewpoints are,  the longer the computational time becomes.  
+<p>
+Parallel computation was implemented exploiting the Python Multiprocessing library and the ability of GRASS GIS to set a temporary spatial region centered on the considered point without affecting the parallel computation of the other points.
+
+
+<h2>EXAMPLES</h2>
+The location used for the following examples can be downloaded from this folder. The name is R.SURVEY_Location
```

Comment:
```suggestion
<h2>EXAMPLES</h2>

The location used for the following examples can be downloaded from this folder. The name is R.SURVEY_Location
```
"This folder" --> please use a link